client/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:20.13.1-alpine
+
+WORKDIR /usr/src/app
+
+RUN npm install -g npm@10.2.4
```

Comment:
This might not be necessary since npm is typically included with Node.js installations. We can remove this line